python/generate/templates/api_client.mustache

```diff
@@ -0,0 +1,731 @@
+# coding: utf-8
```

Comment:
Why is the default_templates file above empty? I have downloaded the file manually to compare against this file so we don't lose time while you add it.

Reply:
Added, great.